webapp/src/app/(dashboard)/[organizationId]/members/page.tsx

```diff
@@ -6,15 +6,15 @@ import { Card } from "@/components/ui/card";
 import { Table, TableBody } from "@/components/ui/table";
 import { Project } from "@/types/project";
 import { User } from "@/types/user";
-import AddMember from "@/components/add-member"
+import AddMember from "@/components/add-member";
```

Comment:
This can be removed !

Reply:
I added it here ==> https://github.com/mlco2/codecarbon/pull/651